File: Latihan B/Python/SoccerTeam.py
class SoccerTeam:

	#private
	nameteam = "0"
	country = "0"
	est = "0"
	nickname = "0"
	player = []

	#konstruksi
	def __init__(self):
		self.nameteam = "0"
		self.country = "0"
		self.est = "0"
		self.nickname = "0"
		self.player = []

	#get set untuk semua variable
	def setNameTeam(self, nameteam):
		self.nameteam = nameteam

	def setCountry(self, country):
		self.country = country

	def setEst(self, est):
		self.est = est

	def setNickname(self, nickname):
		self.nickname = nickname

	def setPlayer(self,  player, idx):
		self.player.insert(idx, player)
		

	

	def getNameTeam(self):
		return self.nameteam
	
	def getCountry(self):
		return self.country

	def getEst(self):
		return self.est
	
	def getPlayer(self, idx):
		return self.player[idx]

	def getNickname(self):
		return self.nickname

File: Latihan B/C++/SoccerTeam.cpp
#include <iostream>
#include <string>
using namespace std;

class SoccerTeam {

	private:
		string nameteam, country, est, nickname , player[50];

	public:
		//konstruksi
		SoccerTeam(){

		}

		//get dan set semua variabel
 
		void setNameTeam(string nameteam){
				this->nameteam = nameteam;
		}
		
		void setCountry(string country){
				this->country = country;
		}

		void setEst(string est){
				this->est= est;
		}

		void setNickname(string nickname){
				this->nickname = nickname;
		}

		void setPlayer(string player, int idx){
				this->player[idx] = player;
		}

		string getNameTeam(){
				return this->nameteam;
		}
		
		string getCountry(){
				return this->country;
		}

		string getEst(){
				return this->est;
		}

		string getNickname(){
				return this->nickname;
		}

		string getPlayer(int idx){
				return this->player[idx];
		}

		//destruktor
		~SoccerTeam(){

		}
};